client/src/main/java/se/fortnox/reactivewizard/client/ConcatenatedJsonObjectsDeserializer.java

```diff
@@ -0,0 +1,123 @@
+package se.fortnox.reactivewizard.client;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.databind.type.TypeFactory;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.core.publisher.Sinks;
+import se.fortnox.reactivewizard.util.ReflectionUtil;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Deque;
+import java.util.LinkedList;
+import java.util.List;
+
+import static java.lang.String.format;
+
+public class ConcatenatedJsonObjectsDeserializer {
```

Comment:
I dont see any reason to introduce support for the client to handle the old streaming {obj},{obj} format. I think we should avoid to add logic to the framework that we actually do not need. If we would add a serialization support in the rw client I think it would be for a format that we do not think is a legacy format.

Reply:
Agreed!